test/e2e_node/apparmor_test.go

```diff
@@ -146,7 +146,9 @@ func loadTestProfiles() error {
 
 func runAppArmorTest(f *framework.Framework, profile string) api.PodStatus {
 	pod := createPodWithAppArmor(f, profile)
-	framework.ExpectNoError(f.WaitForPodNoLongerRunning(pod.Name))
+	// The pod needs to start before it stops, so wait for the longer start timeout.
```

Comment:
not necessarily fail. The pod container just runs touch and exits, and then the apparmor tests look at the container exit code to determine whether it succeeded (which should be conditional on whether the profile was applied correctly)